funnel/templates/proposals.html.jinja2

```diff
@@ -5,9 +5,40 @@
 {% block description %}{{ project.title }}{% if g.profile %} &ndash; {{ g.profile.description|striptags }}{% endif %}{% endblock %}
 
 {%- block pageheaders %}
-  {% assets "css_leaflet" -%}
-    <link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}" />
-  {%- endassets -%}
+  <script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Event",
+    "name": {{ project.title|tojson }},
+    "url": {{ project.url_for(_external=true)|tojson }},
+    "startDate": {% if project.schedule_start_at %}{{ project.schedule_start_at.astimezone(project.timezone).isoformat()|tojson }}{% else %}null{% endif %},
```

Comment:
Apparently `startDate` is mandatory for the event schema, so this `if` condition should be around the entire schema tag. Don't include the schema if there are no scheduled dates.